conf/plants/production_reg_conf.pl

```diff
@@ -30,7 +30,7 @@
 use Bio::EnsEMBL::Registry;
 use Bio::EnsEMBL::Compara::Utils::Registry;
 
-my $curr_release = 98;
+my $curr_release = $ENV{'CURR_ENSEMBL_RELEASE'};
 my $prev_release = $curr_release - 1;
```

Comment:
Since we are automatising the change between releases, I would add these two new variables:
`my $curr_eg_release = $curr_release - 53;`
`my $prev_eg_release = $curr_eg_release - 1;`